wachter/actions.py

```diff
@@ -1,31 +1,33 @@
 import json
 import logging
-import telegram
-from telegram import InlineKeyboardButton, InlineKeyboardMarkup
+from telegram import Bot, Update, Message, InlineKeyboardButton, InlineKeyboardMarkup, ParseMode
+from telegram.ext import Job, JobQueue
+from telegram.error import TelegramError
 from datetime import datetime, timedelta
 from model import Chat, User, session_scope, orm_to_dict
 from constants import Actions, RH_kick_messages, RH_CHAT_ID
 import constants
 import re
 import random
+from typing import int, dict
```

Comment:
`int` and `dict` are not parts of `typing` -- those are native python types, let's use them without the import